nodl_python/nodl/_util.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it under the terms of the
+# GNU Limited General Public License version 3, as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranties of MERCHANTABILITY, SATISFACTORY QUALITY, or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Limited General Public License for more details.
+#
+# You should have received a copy of the GNU Limited General Public License along with
+# this program. If not, see <http://www.gnu.org/licenses/>.
+
+
+def str_to_bool(boolean_string: str) -> bool:
+    """Convert xml boolean entries to `bool`."""
+    return boolean_string.lower() in ['true', '1']
```

Comment:
Given the function naming and because you are looking for a match in a list of strings, I'll go all the way down and search in both a 'true' list and a 'false' list and throw otherwise. As of now `str('trololo')` evaluates to false.

Reply:
5c312d3